cmd/operator-sdk/bundle/create.go

```diff
@@ -78,7 +78,9 @@ $ operator-sdk bundle create \
 				if len(args) != 0 {
 					return fmt.Errorf("command %s does not accept any arguments", cmd.CommandPath())
 				}
-				err := bundle.GenerateFunc(c.directory, c.packageName, channels,
+
+				outputDir := filepath.Join(c.directory,"/metadata")
+				err := bundle.GenerateFunc(c.directory, outputDir, c.packageName, channels,
```

Comment:
We need to upgrade the Operator Registry was well and it is asking for the outputdir too.  
c/c @estroz

Reply:
Following the test performed locally.

- The command was executed with success. 

```
$ operator-sdk bundle create --generate-only
INFO[0000] Building annotations.yaml                    
INFO[0000] Generating output manifests directory        
INFO[0000] Building Dockerfile                          
```

- The files were gen in the dir.

```
camilamacedo@Camilas-MacBook-Pro ~/go/src/github.com/operator-framework/operator-sdk-samples/go/memcached-operator/deploy/olm-catalog/memcached-operator/metadata (master) $ ls -la
total 0
drwxr-xr-x   4 camilamacedo  staff  128 31 Mar 17:07 .
drwxr-xr-x  13 camilamacedo  staff  416 31 Mar 17:07 ..
drwxr-xr-x   4 camilamacedo  staff  128 31 Mar 17:07 manifests
drwxr-xr-x   3 camilamacedo  staff   96 31 Mar 17:07 metadata
```

- See:

```
$ ls
memcached-operator		memcached-operator.package.yaml
```

```
$ cat memcached-operator.package.yaml 
channels:
- currentCSV: memcached-operator.v0.15.2
  name: alpha
defaultChannel: alpha
packageName: memcached-operator
```

And ;

```
o ~/go/src/github.com/operator-framework/operator-sdk-samples/go/memcached-operator/deploy/olm-catalog/memcached-operator/metadata/manifests/memcached-operator (master) $ ls -la
total 96
drwxr-xr-x  12 camilamacedo  staff   384 31 Mar 17:07 .
drwxr-xr-x   4 camilamacedo  staff   128 31 Mar 17:07 ..
-rw-r--r--   1 camilamacedo  staff  1991 31 Mar 17:07 cache.example.com_memcacheds_crd.yaml
-rw-r--r--   1 camilamacedo  staff  3196 31 Mar 17:07 memcached-operator.v0.10.0.clusterserviceversion.yaml
-rw-r--r--   1 camilamacedo  staff  5066 31 Mar 17:07 memcached-operator.v0.11.0.clusterserviceversion.yaml
-rw-r--r--   1 camilamacedo  staff  5027 31 Mar 17:07 memcached-operator.v0.12.0.clusterserviceversion.yaml
-rw-r--r--   1 camilamacedo  staff  3821 31 Mar 17:07 memcached-operator.v0.13.0.clusterserviceversion.yaml
-rw-r--r--   1 camilamacedo  staff  3821 31 Mar 17:07 memcached-operator.v0.14.0.clusterserviceversion.yaml
-rw-r--r--   1 camilamacedo  staff  3821 31 Mar 17:07 memcached-operator.v0.15.2.clusterserviceversion.yaml
-rw-r--r--   1 camilamacedo  staff  3866 31 Mar 17:07 memcached-operator.v0.16.0.clusterserviceversion.yaml
-rw-r--r--   1 camilamacedo  staff  3083 31 Mar 17:07 memcached-operator.v0.5.0.clusterserviceversion.yaml
-rw-r--r--   1 camilamacedo  staff  3121 31 Mar 17:07 memcached-operator.v0.6.0.clusterserviceversion.yaml
```

So, IMO the